R/install.R

```diff
@@ -44,6 +56,45 @@ safe_install_packages <- function(...) {
   )
 }
 
+safe_build_package <- function(pkgdir, build_opts, dest_path, quiet, use_pkgbuild = is_installed("pkgbuild")) {
+  if (use_pkgbuild) {
+    vignettes <- TRUE
+    manual <- FALSE
+    has_no_vignettes <- grepl("--no-build-vignettes", build_opts)
+    if (any(has_no_vignettes)) {
+      vignettes <- FALSE
+    }
+    has_no_manual <- grepl("--no-manual", build_opts)
+    if (!any(has_no_manual)) {
+      manual <- TRUE
+    }
+    build_opts <- build_opts[!(has_no_vignettes | has_no_manual)]
+    pkgbuild::build(pkgdir, dest_path = dest_path, binary = FALSE,
+      vignettes = vignettes, manual = manual, args = build_opts, quiet = quiet)
+  } else {
+    # No pkgbuild, so use tools:::.build_packages, which is what is called by R
+    # CMD build. Calling it directly avoids needing to start a new R CMD build
+    # process (which varies across OSs) and deal with shell quoting (which also
+    # varies across OSs)
```

Comment:
This is not actually what the code does. Maybe these comments are wrong?

Reply:
Sorry yes, forgot to remove these